### ISSUE TITLE
http://shadowsocks.org/ubuntu no longer available

### ISSUE THREAD

**Issue by nebulakl** [jlund/streisand#517]
now if u try to access http://shadowsocks.org/ubuntu will result in 404, which interrupts streisand's installation. Is is possible to use https://github.com/shadowsocks/shadowsocks-libev instead? (i.e. download source then compile instead of using a precompiled version)

**Comment by sjug** [comment]
Duplicate of https://github.com/jlund/streisand/issues/507 and https://github.com/jlund/streisand/issues/510. 
@jlund you need a co-maintainer who is active...

**Comment by dlin303** [comment]
Also having the same issue. I see a lot of suggestions for removing shadowsocks from the streisand setup, but I'd like to keep shadowsocks so not an option for me.

**Comment by jlund** [comment]
Hey, everyone! I just pushed a fix for this. I am really sorry about the delay. Both my work and personal life have been pretty crazy lately and I was not expecting this repository to disappear :)

Shadowsocks is now being compiled from source while I investigate a long-term replacement strategy. Things will either be moving to ShadowsocksR or Shadowsocks2. This is being tracked in issue #501.

Thanks for your patience!

**Comment by jlund** [comment]
@sjug Point taken. This week was bad timing for the Shadowsocks repo to vanish and for me to have a packed schedule where I couldn't check my personal email. I think my track record demonstrates fast response and reaction times over the life of the project, but I also don't want to feel like a single point of failure. I have recruited some co-maintainers (the most recent two were added only a few months ago), but their time recently also appears limited.

I'm in the middle of a pretty big refactor to enable role selection (e.g. choosing which services you want by commenting out undesired roles) and once this is done I will probably focus on finding additional participants. I want the project to remain healthy and this incident has made me sad.

Happy Valentine's Day!

**Comment by sjug** [comment]
@jlund I totally understand, you just have a very important project to a lot of people. So even if you do have life come up (which always will) hopefully you can have people to keep it running smoothly.

I've taken a look at separating out the services myself and that's a huge refactor, but would be a huge feature as well. Are you still taking PRs, as I'd be interested in contributing but I don't know what your roadmap is. Are you available to talk via some medium?

**Comment by jlund** [comment]
Yeah, I'm on Signal and WhatsApp. We can exchange details using email or Twitter DMs--or let me know if you would prefer some other medium.

**Comment by sjug** [comment]
@jlund Signal is ideal yes, where can I find your email?

**Comment by jlund** [comment]
The one in `git log` is my primary address. It is kind of naive that I'm not just writing it here, but maybe there are some spambots out there that haven't already harvested it :)

**Comment by sjug** [comment]
Ah good one
